diff --git a/android/build.gradle b/android/build.gradle
index 0000000..1111111 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -51,7 +51,7 @@ android {
 }
 
 android {
-  compileSdkVersion safeExtGet("compileSdkVersion", 34)
+  compileSdk safeExtGet("compileSdkVersion", 34)
 
   compileOptions {
@@ -69,7 +69,7 @@ android {
     versionName "0.1.0"
   }
-  lintOptions {
+  lint {
     abortOnError false
   }
   publishing {
